Dockerfile

```diff
@@ -0,0 +1,34 @@
+from python:3.9-slim
+
+# Generate locale C.UTF-8 for postgres and general locale data
+ENV LANG C.UTF-8
+ARG ODOO_BRANCH=14.0
+
+# Install dependencies
+RUN apt update && apt install -y git npm postgresql-client python3-dev libxml2-dev libxslt1-dev libldap2-dev libsasl2-dev \
+    libtiff5-dev zlib1g-dev libfreetype6-dev wait-for-it xvfb libfontconfig wkhtmltopdf \
+    liblcms2-dev libwebp-dev libharfbuzz-dev libfribidi-dev libxcb1-dev libpq-dev gettext-base unzip
+
+RUN pip install phonenumbers
+RUN npm install -g rtlcss
+RUN apt install -y wget gettext-base
+
+RUN mkdir -p /opt/odoo
+RUN cd /tmp/ && wget https://github.com/mekomsolutions/odoo/archive/refs/heads/${ODOO_BRANCH}.zip \
+    && unzip /tmp/${ODOO_BRANCH}.zip && mv /tmp/odoo-${ODOO_BRANCH}/* /opt/odoo/ && rm /tmp/${ODOO_BRANCH}.zip
+RUN cd /opt/odoo && pip install -r requirements.txt
+
+# Expose Odoo services
+EXPOSE 8069 8071 8072
+VOLUME ["/var/lib/odoo", "/mnt/extra-addons"]
```

Comment:
I don't think this should be set as a volume as such.
Volumes there are meant for data.

This one however is an optional bind volume that one can provide if willing to provide addons.
Documentation should suffice to tell this.